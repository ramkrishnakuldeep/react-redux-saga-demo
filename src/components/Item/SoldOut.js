import React from 'react';
import './sold_out.scss';

const SoldOut = () => {
    return (
        <div className="sold-out">
            <div className="triangle"> </div>
            <span> SOLD </span>
        </div>
    );
};

export default SoldOut;
