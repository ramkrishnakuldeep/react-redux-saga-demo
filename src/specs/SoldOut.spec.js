import React from 'react';
import { shallow } from '../enzyme';
import SoldOut from '../components/Item/SoldOut';

describe('SoldOut component', () => {
    it('initial render', () => {
        const component = shallow(<SoldOut />);
        expect(component).toMatchSnapshot();
    });
});
