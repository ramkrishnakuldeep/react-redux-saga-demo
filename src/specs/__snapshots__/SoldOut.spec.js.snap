// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SoldOut component initial render 1`] = `
<div
  className="sold-out"
>
  <div
    className="triangle"
  >
     
  </div>
  <span>
     SOLD 
  </span>
</div>
`;
